src/main/java/cms/gov/madie/measure/models/Measure.java

```diff
@@ -26,9 +26,9 @@
   private String state;
 
   @Indexed(unique = true)
-  @NotBlank(
-      groups = {ValidationOrder1.class},
-      message = "Measure Library Name is required")
+  // @NotBlank(
+  // groups = {ValidationOrder1.class},
+  // message = "Measure Library Name is required")
```

Comment:
we can leave it optional but this works with the changes that I merged in for cqlLibraryName